elements/pf-avatar/BaseAvatar.ts

```diff
@@ -28,15 +28,17 @@ export class BaseAvatar extends LitElement {
   /** Size of the Avatar */
   @property({ reflect: true }) size: 'sm'|'md'|'lg'|'xl' = 'sm';
 
+  /** Whether or not the Avatar image is dark */
+  @property({ type: Boolean, reflect: true, attribute: 'isdark' }) isDark = false;
+
   render() {
-    const isDark = false; // TODO: later
     return this.src != null ? html`
       <img
         size=${this.size}
         alt=${this.alt ?? ''}
         @load="${(e: Event) => this.dispatchEvent(new AvatarLoadEvent(e))}"
         src=${this.src}>
-    ` : isDark ? html`
+    ` : this.isDark ? html`
       <svg xmlns="http://www.w3.org/2000/svg" style="enable-background:new 0 0 36 36" viewBox="0 0 36 36">
         <style>.st1,.st2{fill-rule:evenodd;clip-rule:evenodd;fill:#6a6e73}.st2{fill:#4f5255}</style><path d="M0 0h36v36H0z" style="fill:#212427"/>
```

Comment:
this is fine, but it would be better to move the styles to svg attributes

even better would be to change the attribute values using the `svg` template tag, rather than switching off of `dark` and returning a completely new template.

there is an experimental `spread` helper but it's maybe not yet ready for production